lib/splicer.rb

```diff
@@ -101,4 +112,13 @@ def self.delete_record_in_zone(record, zone)
   def self.logger
     @@logger || NullObject.new
   end
+
+  def self.arguments_are_invalid(record, zone)
```

Comment:
User a more constructive name for this method. `arguments_are_invalid` could mean anything.